### ISSUE TITLE
Gramophone Factory : productionTime

### ISSUE THREAD

**Issue by Arkentus** [suhrmann/Anno-1800-Calculator#20]
```
 "Gramophone Factory": {
            "product": "Gramophones",
            "building": "Gramophone Factory",
            "img": "investors/gramophones.webp",
            "worldID": 1,
            "productionTime": 120,
```

**productionTime -> 60**

![gramophone](https://user-images.githubusercontent.com/25131967/63647383-ca64df00-c720-11e9-8b25-01bbbe6230e5.png)

**Comment by tluderer** [comment]
Hey, sorry for the delay - holiday season :)
Since the 120s are still on the wiki page, i will check the value ingame as well in the next days.
But since you sent a screenshot, i will change the data for now

**Comment by tluderer** [comment]
fixed with commit 3b40228d2f84cf0f126d259180dbc9734c63fac2
